src/test/java/Pages/RegistrationPage.java

```diff
@@ -0,0 +1,16 @@
+package Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class RegistrationPage extends BasePage {
+    public RegistrationPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    By registrationBtn = By.cssSelector("a[href='registration']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```